.github/workflows/publish_android_maven_central.yml

```
@@ -0,0 +1,50 @@
+name: Publish Capacitor Plugin to Maven Central
+
+on:
+  workflow_call:
+    secrets:
+      ANDROID_CENTRAL_USERNAME:
+        required: true
+      ANDROID_CENTRAL_PASSWORD:
+        required: true
+      ANDROID_SIGNING_KEY_ID:
+        required: true
+      ANDROID_SIGNING_PASSWORD:
+        required: true
+      ANDROID_SIGNING_KEY:
+        required: true
+      ANDROID_SONATYPE_STAGING_PROFILE_ID:
+        required: true
+      CAP_GH_RELEASE_TOKEN:
+        required: true
+  workflow_dispatch:
+
+jobs:
+  publish-android:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    permissions:
+      contents: read
+      packages: write
```

From merging my PR, just realised this is incorrect. We won't publish to GitHub packages so we shouldn't have the permission (was actually causing the pipeline to fail because the permission was not declared in `release_plugin.yml`)

```suggestion
    permissions:
      contents: read
```